src/index.rs

```diff
@@ -339,7 +339,7 @@ impl Index {
         .begin_read()?
         .open_table(ORDINAL_TO_INSCRIPTION)?
         .get(&ordinal.n())?
-        .map(|inscription| Inscription(inscription.to_owned())),
```

Comment:
Add context and return the error. Something like "Failed to deserialize inscription".